pkg/liquidity-mining/contracts/GaugeWorkingBalanceHelper.sol

```diff
@@ -0,0 +1,144 @@
+// SPDX-License-Identifier: GPL-3.0-or-later
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+pragma solidity ^0.7.0;
+
+import "@balancer-labs/v2-interfaces/contracts/solidity-utils/openzeppelin/IERC20.sol";
+import "@balancer-labs/v2-interfaces/contracts/liquidity-mining/IVeDelegation.sol";
+
+import "@balancer-labs/v2-solidity-utils/contracts/math/FixedPoint.sol";
+import "@balancer-labs/v2-solidity-utils/contracts/math/Math.sol";
+
+interface IVeDelegationProxy is IVeDelegation {
+    function getVotingEscrow() external view returns (IERC20);
+}
+
+interface IGauge {
+    function balanceOf(address user) external view returns (uint256);
+
+    function totalSupply() external view returns (uint256);
+
+    // solhint-disable func-name-mixedcase
+    function working_balances(address user) external view returns (uint256);
+
+    function working_supply() external view returns (uint256);
+    // solhint-enable func-name-mixedcase
+}
```

Comment:
It's a one-off interface, only used in this helper. Since it can't really be shared or reused, and this is a helper function, I thought it was clearer to have it self-contained.

We do this elsewhere (e.g., in the vendor-specific Linear Pools, when we need to call functions on external non-Balancer contracts)